stl/inc/bit

```diff
@@ -253,9 +253,9 @@ _NODISCARD int _Checked_arm_arm64_countl_zero(const _Ty _Val) noexcept {
     }
 #else // ^^^ workaround / no workaround vvv
     if constexpr (_Digits <= 32) {
-        return static_cast<int>(_CountLeadingZeros(_Val));
+        return static_cast<int>(_CountLeadingZeros(_Val)) - (numeric_limits<unsigned long>::digits - _Digits);
     } else {
-        return static_cast<int>(_CountLeadingZeros64(_Val));
+        return static_cast<int>(_CountLeadingZeros64(_Val)) - (numeric_limits<unsigned long long>::digits - _Digits);

```

Comment:
This looks misleading. There are no types that are larger than 32 bits but smaller than 64.

Reply:
Also the _workaround_ path only subtracts for `<= 32` case too

Reply:
I agree, this should be dropped.